src/style.css

```diff
@@ -0,0 +1,132 @@
+@font-face {
+  font-family: "Avenir-Book";
+  src: url("fonts/Avenir-Book.ttf");
```

Comment:
Implement this task requirement 
![image](https://github.com/mate-academy/layout_search-bar-airbnb/assets/70009205/421f1a90-3a30-4971-8ec8-c6ae9517f4f0)

```suggestion
  src: url("fonts/Avenir-Book.ttf");
  font-weight: 300;
```